lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -68,21 +77,30 @@ export class PublicClientApplication implements IPublicClientApplication {
      * @param IController Optional parameter to explictly set the controller. (Will be removed when we remove public constructor)
      */
     public constructor(configuration: Configuration, controller?: IController) {
+        this.configuration = configuration;
         if (controller) {
             this.controller = controller;
         } else {
-            const standardOperatingContext = new StandardOperatingContext(
-                configuration
+            const operatingContext = new UnknownOperatingContext(configuration);
+            this.controller = new UnknownOperatingContextController(
+                operatingContext
             );
-            this.controller = new StandardController(standardOperatingContext);
         }
     }
 
     /**
      * Initializer function to perform async startup tasks such as connecting to WAM extension
      */
     async initialize(): Promise<void> {
-        return this.controller.initialize();
+        if (this.controller instanceof UnknownOperatingContextController) {
+            const factory = new ControllerFactory(this.configuration);
```

Comment:
Ah so the StandardController gets built in initialize instead? How does that work for APIs that don't require initialize (getAllAccounts for instance) that may be invoked before initialize?

Reply:
The default controller is UnknownOperatingContextController... and it has the implementation for those methods.